src/java/org/jivesoftware/openfire/plugin/MonitoringPlugin.java

```diff
@@ -170,28 +173,22 @@ public void initializePlugin(PluginManager manager, File pluginDirectory) {
         xep0313Support2 = new Xep0313Support2(XMPPServer.getInstance());
         xep0313Support2.start();
 
-        // Check if we Enterprise is installed and stop loading this plugin if
-        // found
-        File pluginDir = new File(JiveGlobals.getHomeDirectory(), "plugins");
-        File[] jars = pluginDir.listFiles(new FileFilter() {
-            public boolean accept(File pathname) {
-                String fileName = pathname.getName().toLowerCase();
-                return (fileName.equalsIgnoreCase("enterprise.jar"));
-            }
-        });
-        if (jars.length > 0) {
+        // Check if we Enterprise is installed and stop loading this plugin if found
+        if (manager.getPluginByName("enterprise").isPresent()) {
```

Comment:
Is this still necessary?
Would this plugin be compatible with 4.8?

Reply:
I've removed this bit. Good catch.